Content.Server/GameObjects/Components/Items/Storage/ServerStorageComponent.cs

```diff
@@ -48,6 +58,7 @@ public class ServerStorageComponent : SharedStorageComponent, IInteractUsing, IU
 
         [ViewVariables]
         public override IReadOnlyList<IEntity>? StoredEntities => _storage?.ContainedEntities;
+        public string _soundCollectionName = "storageRustle";
```

Comment:
Why isn't this serialized? And it should probably default to no sound collection, in my opinion